docs/includes/yeoman-generator-recommendation.md

```diff
@@ -0,0 +1,6 @@
+> [!TIP]
```

Comment:
Where else is this recomendation getting added?

Reply:
It doesn't appear anywhere else yet -- but will soon, when I revamp the [Create and debug Office Add-ins in Visual Studio](https://docs.microsoft.com/en-us/office/dev/add-ins/develop/create-and-debug-office-add-ins-in-visual-studio) article per work item [3711752](https://office.visualstudio.com/OC/_queries/edit/3711752). Figured I'd make it an include now, to save the trouble of having to do so later.